src/js/classes.js

```diff
@@ -269,22 +266,22 @@ Classes.CLASSES = {
   'Trickster': {
     spec_type: 'Luck',
     characters: {
-      'Ein': {
-        title: 'The Gambler',
+      ein: {
```

Comment:
Sorry, could you revert this? (Or was it intentional? If so, i might need to verify the code still works)